packages/react-router-dev/typedoc.json

```diff
@@ -3,6 +3,7 @@
     "./index.ts",
     "./routes.ts",
     "./vite.ts",
-    "./vite/cloudflare.ts"
+    "./vite/cloudflare.ts",
+    "./config.ts"
```

Comment:
Heads up that this has already been added in https://github.com/remix-run/react-router/pull/12332.